src/templates/partials/traefik.hbs

```diff
@@ -0,0 +1,36 @@
+# generated with: {{{output.link}}}
+[entryPoints]
+[entryPoints.http.tls]
+{{#if (includes "modern" form.config)}}
+minVersion = "VersionTLS12"
+{{else if (includes "intermedia" form.config)}}
+minVersion = "VersionTLS10"
+{{else}}
+minVersion = "VersionSSL30"
+{{/if}}
+
+cipherSuites = [
```

Comment:
You'll want to use a loop over `output.cipherSuites` to list your cipher suites, instead of hard codingf them. Take a look at how `awselb.hbs` does it to get a good idea of how it works.  :)

Reply:
A simple loop doesn't work because the golang constants that Traefik uses are very different from what other templates use so far.

https://godoc.org/crypto/tls#pkg-constants

The internet tells me to check the openssl constants and use them: https://www.woolie.co.uk/article/mapping-openssl-ciphers-to-iana-cipher-suite-registy/, but those aren't exposed currently.

Any suggestions on what route to take here?

Reply:
@april Any suggestions?